wifiphisher/data/phishing-pages/oauth-login/html/static/css/style2.css

```diff
@@ -0,0 +1,217 @@
+body {
```

Comment:
Why there is two style.css : The first one is for the popup page (login.html) and the second one is for the default oauth scenario if I used only one sytle.css it will be some conflicts in both pages. I'll rename the second one.

Reply:
Yes, if you want to have two, we need a more descriptive name (e.g. `style-popup.css` and `style-form.css`)

Reply:
That's a good idea. I'll do it on the next update.